internal/cmd/testdata/scripts/templatefuncs_windows.txtar

```diff
@@ -0,0 +1,9 @@
+[!windows] skip 'Windows only'
+
+# test lookPathIn template function to find in specified script - success
+exec chezmoi execute-template '{{ lookPathIn "winver" "c:\windows" }}'
+stdout ^/bin/echo
+
+# test lookPathIn template function to find in specified script - failure
+exec chezmoi execute-template '{{ lookPathIn "winver" "c:\users" }}'
```

Comment:
This probably also needs another backslash.

Reply:
Yep you're right. I went in blind. I have identified an actual issue too with this test.